.github/workflows/ci.yml

```
@@ -0,0 +1,17 @@
+name: Tests and Style
+on: [push, pull_request]
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
+        ruby: [2.5, 2.6, 2.7, '3.0', head]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+    - run: bundler exec rake && bundler exec rubocop
```

I'd split the command to more steps to make the action log better readable:

```suggestion
    - name: Unit Tests 
      run: bundler exec rake
    - name: Rubocop
      run: bundler exec rubocop
```